container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/user_services_functions/copy_files_from_user_service.go

```diff
@@ -60,15 +64,27 @@ func CopyFilesFromUserService(
 	srcPath = filepath.Clean(srcPath)
 	// we get the base dir | file
 	srcPathBase := filepath.Base(srcPath)
-	// we get the dir that holds base the dir | file
+	//// we get the dir that holds base the dir | file
 	srcPathDir := filepath.Dir(srcPath)
 
-	commandToRun := fmt.Sprintf(
-		`if command -v 'tar' > /dev/null; then cd '%v' && tar cf - '%v'; else echo "Cannot copy files from path '%v' because the tar binary doesn't exist on the machine" >&2; exit 1; fi`,
-		srcPathDir,
-		srcPathBase,
-		srcPath,
-	)
+	var commandToRun string
+
+	if srcPathBase == doNotIncludeParentDirInArchiveSymbol {
+		commandToRun = fmt.Sprintf(
+			commandString,
+			srcPathDir,
+			ignoreParentDirInArchiveSymbol,
+			srcPath,
+		)
+	} else {
+		commandToRun = fmt.Sprintf(
+			commandString,
+			srcPathDir,
+			srcPathBase,
+			srcPath,
+		)
+	}
```

Comment:
```suggestion
	if srcPathBase == doNotIncludeParentDirInArchiveSymbol {
		srcPathBase = ignoreParentDirInArchiveSymbol
	}
	commandToRun = fmt.Sprintf(
		commandString,
		srcPathDir,
		srcPathBase,
		srcPath,
	)
```

Reply:
ah man; that's neater - didn't even realized it. 💯